numpy/core/src/umath/ufunc_object.c

```diff
@@ -479,8 +480,26 @@ _is_alnum_underscore(char ch)
     return _is_alpha_underscore(ch) || (ch >= '0' && ch <= '9');
 }
 
+NPY_NO_EXPORT npy_longlong
+npy_strtoll(const char *str, char **endptr, int base);
```

Comment:
Would prefer that we move this to a header, rather than repeating the prototype

Reply:
leftover cruft removed